index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Weather Report</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+    <script src="scripts/index.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    
+</head>
+<body>
+    <h1 id="h1t">Weather Report</h1>
+    <h2 id="city_name_display">For the city of:</h2>
```

Comment:
You can add a default city to this header just so it's not empty 

```suggestion
    <h2 id="city_name_display">For the city of: Seattle</h2>
```